api/client.go

```diff
@@ -221,7 +221,7 @@ func (c Client) StartDeviceFlow() (*DeviceFlowResponse, error) {
 	return post[EmptyRequest, DeviceFlowResponse](c, "/api/device", nil)
 }
 
-func (c Client) PollDeviceFlow(req *PollDeviceFlowRequest) (*DevicePollResponse, error) {
+func (c Client) GetDeviceFlowStatus(req *PollDeviceFlowRequest) (*DevicePollResponse, error) {
```

Comment:
I renamed this to be consistent with the API handler name, and to better match the URL. The name `Poll` could suggest that this call itself does polling.

Reply:
Good catch. I renamed the request type to `DeviceFlowStatusRequest` and response to `DeviceFlowStatusResponse`. After those renames I can't find anything else that using `PollDeviceFlow`